### PR TITLE
Fix #8 app logo issue fixed

### LINKED ISSUE #8: Np-chat app logo doesnt appear properly
The logo is not properly added to mipmap to make it compatible with all the devices.

screenshot_20190303-210138

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #8 

New mipmap assets added

screenshot_20190303-213721

### CODE REVIEW

**Review by agawande — CHANGES_REQUESTED** [review]
Need to add app/app.iml to gitignore. Otherwise looks good!

**Comment by neel1998** [comment]
@agawande i have made the requested changes , kindly review

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@ build/
 gen/
 .gradle
 local.properties
-
+app/app.iml
```
Comment (agawande) [inline .gitignore]: Can go under *.iml below. Also please delete the existing app/app.iml file from git.

**Comment by neel1998** [comment]
@agawande  what do you mean by deleting the file from git?

**Comment by agawande** [comment]
> @agawande what do you mean by deleting the file from git?

So it is no longer shows up in the repository (i.e. not tracked by git any longer).
(iirc git rm --cached app/app.iml)

**Comment by neel1998** [comment]
@agawande  should i do it on master branch or this branch only and then you will merge?

**Comment by agawande** [comment]
> @agawande should i do it on master branch or this branch only and then you will merge?

This branch only, then I will merge.

**Comment by neel1998** [comment]
@agawande  done. Please check

**Review by agawande — COMMENTED** [review]
Just one more small change - to put app/app.iml under the IntelliJ comment in .gitignore

**Comment by neel1998** [comment]
@agawande  done

**Pull request merged**